tests/test_database.py

```diff
@@ -23,6 +25,45 @@ def create_test_database_setup(database_connector):
          "ItemId VARCHAR(45) NOT NULL);"))
 
 
+def count_entries(database_connector, table_name):
+    sql_query = "SELECT COUNT(*) FROM " + table_name
+    count = database_connector.execute(sql_query)
+    return count[0]['COUNT(*)']
+
+
+def test_items_tags_setup(test_item_manager, test_tag_manager):
+    item_ids = []
+
+    item_id = database_helper.get_id()
+    item_ids.append(item_id)
+    test_item_manager.add_item(item_id)
+    test_tag_manager.add_tag("bird", item_id)
+
+    item_id = database_helper.get_id()
+    item_ids.append(item_id)
+    test_item_manager.add_item(item_id)
+    test_tag_manager.add_tag("bird", item_id)
+
+    item_id = database_helper.get_id()
+    item_ids.append(item_id)
+    test_item_manager.add_item(item_id)
+    test_tag_manager.add_tag("bird3", item_id)
+
+    item_id = database_helper.get_id()
+    item_ids.append(item_id)
+    test_item_manager.add_item(item_id)
+    test_tag_manager.add_tag("h", item_id)
+    test_tag_manager.add_tag("hiya", item_id)
+    test_tag_manager.add_tag("hello", item_id)
+    return item_ids
+
+
+def is_tag_in_result(tag, result):
+    for val in result:
+        if val['Tag'] == tag:
+            return True
+
```

Comment:
Perhaps add `return False`, even though None is falsy

Reply:
added false return